backend/src/main/java/de/neuefische/backend/service/EventService.java

```diff
@@ -0,0 +1,38 @@
+package de.neuefische.backend.service;
+
+import de.neuefische.backend.model.Event;
+import de.neuefische.backend.repo.EventRepo;
+import org.springframework.data.domain.Example;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+public class EventService {
+
+    private final EventRepo eventRepo;
+
+    public EventService(EventRepo eventRepo) {
+        this.eventRepo = eventRepo;
+    }
+
+    public Event addEvent(Event event) {
+        String id = UUID.randomUUID().toString();
```

Comment:
you could extract the generation of the id to an own class for easier testing

Reply:
My suggestion would be to delete the ID generation here and let the database generate the ID.